managers/elasticsearch.py

```diff
@@ -29,14 +29,32 @@ def createElasticConnection(self, scheme=None, host=None, port=None, user=None,
 
         creds = '{}:{}@'.format(user, pswd) if user and pswd else ''
 
-        host = '{}://{}{}:{}'.format(scheme, creds, host, port)
+        multHosts = []
+
+        if isinstance(host, list) or \
+            isinstance(host, dict) or \
+            isinstance(host, set) or \
+            isinstance(host, tuple):
```

Comment:
minor improvement here `isinstance` can take a tuple of types to check against!

Reply:
I'll do just that.